app/views/schools/ects/teacher_leaving_wizard/check_answers.html.erb

```diff
@@ -0,0 +1,26 @@
+<% page_data(
+    title: "Confirm that #{@wizard.teacher_full_name} has left your school permanently",
+    backlink_href: @wizard.previous_step_path
+) %>
+
+<p class="govuk-body">
+  You’ve told us that <%= @wizard.teacher_full_name %> 
+  left your school on <%= @wizard.current_step.leaving_date.to_fs(:govuk) %>.
```

Comment:
Does this copy need to be updated if the leaving date is in the future?

Reply:
I've added logic for this in #1884